src/app/pages/tables/components/hotTables/handsontable/science-demo.ts

```diff
@@ -0,0 +1,73 @@
+import { Component } from '@angular/core';
+import * as Handsontable from 'handsontable/dist/handsontable.full.js';
+import { getScienceData } from './data';
+
+let heatmapScale = chroma.scale(['#17F556', '#ED6D47']);
+let heatmap = [];
+
+function updateHeatmap(change, source) {
```

Comment:
@daoudaDiallo please, move the external functions (updateHeatmap, point, generateHeatmapData) to a new service.